app/styles/ember-rdfa-editor/_c-content.scss

```diff
@@ -123,17 +135,17 @@ $say-editor-highlight-selected-color: var(--au-gray-300) !default;
   }
 
   // Setup some margin rules
-  p,
-  ul,
-  ol,
-  table,
+  p.say-paragraph,
+  ul.say-bullet-list,
+  ol.say-ordered-list,
+  table.say-table,
   div,
```

Comment:
Would it be possible to replace this `div` by e.g. `.say-block-rdfa`. This way that rule would be less general, and wouldn't target unwanted elements.